ArchiveScraping/targetSpeaker.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+'''Strips description column for relevant information'''
+def findSpeaker():
+    try:
```

Comment:
Doc strings should go below the def line (and indented). My mistake. I was using this pattern a few commits back in master and updated it recently. Get the most up to date version with a git fetch/pull.
This is according to the flake-8 standard. If you install the linter listed in the 'tooling' wiki page, it will give you little warnings on the side as you type your code.
Also make your commits smaller. One line (or a few line) diffs is fine, even preferred. Adding doc strings should have had its own commit. It's easier to review since I can look back at each commit you made and it will make the git blame much cleaner when we have a larger codebase with more history.
Not to mention your git history (green boxes on your profile page) look much more 'productive' to employers.